pkg/formatters/detailsFormatter.go

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package formatters
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/galasa.dev/cli/pkg/galasaapi"
+)
+
+// -----------------------------------------------------
+// Detailed format.
+const (
+	DETAILS_FORMATTER_NAME = "details"
+)
+
+type DetailsFormatter struct {
+}
+
+func NewDetailsFormatter() RunsFormatter {
+	return new(DetailsFormatter)
+
+}
+
+func (*DetailsFormatter) GetName() string {
+	return DETAILS_FORMATTER_NAME
+}
+
+func (*DetailsFormatter) FormatRuns(runs []galasaapi.Run, apiServerUrl string) (string, error) {
+	var result string = ""
+	var err error = nil
+
+	if len(runs) < 1 {
+		return result, err
+	}
+
+	buff := strings.Builder{}
+
+	for i, run := range runs {
+		var duration string
+		var startTimeString string
+		var endTimeString string
+		startTimeStringRaw := run.TestStructure.GetStartTime()
+		endTimeStringRaw := run.TestStructure.GetEndTime()
+
+		if len(startTimeStringRaw) > 0 {
+			startTimeString = formatTime(startTimeStringRaw)
+			if len(endTimeStringRaw) > 0 {
+				endTimeString = formatTime(endTimeStringRaw)
+			}
+		}
+
+		startTime, err := time.Parse("2006-01-02 15:04:05", startTimeString)
```

Comment:
can this constant be a symbolic rather than a literal that gets used several times pls ?